### PR TITLE
[New Rule] Privilege Escalation via Rogue Named Pipe Impersonation

### PR DESCRIPTION
**Pull request by Samirbous**
Resolves https://github.com/elastic/detection-rules/issues/1543

### CODE REVIEW

File: rules/windows/privilege_escalation_via_rogue_named_pipe.toml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Identifies a privilege escalation attempt via rogue named pipe impersonation. An adversary may abuse this technique by
+masquerading as a known named pipe and manipulating a privileged process to connect to it.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Privilege Escalation via Rogue Named Pipe Impersonation"
+note = """Namped Pipe Creation Events need to be enabled in you Sysmon configuration by including the following settings
+condition equal "contains" and keyword equal "pipe".
```
Comment (w0rk3r) [inline rules/windows/privilege_escalation_via_rogue_named_pipe.toml:18]:
```suggestion
note = """Named Pipe Creation Events need to be enabled in you Sysmon configuration by including the following settings
condition equal "contains" and keyword equal "pipe".
```

File: rules/windows/privilege_escalation_via_rogue_named_pipe.toml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+masquerading as a known named pipe and manipulating a privileged process to connect to it.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Privilege Escalation via Rogue Named Pipe Impersonation"
+note = """Named Pipe Creation Events need to be enabled in you Sysmon configuration by including the following settings
+condition equal "contains" and keyword equal "pipe".
+"""
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_via_rogue_named_pipe.toml:19]:
```suggestion
note = """## Config

Named Pipe Creation Events need to be enabled within the Sysmon configuration by including the following settings:
`condition equal "contains" and keyword equal "pipe"`
"""
```

File: rules/windows/privilege_escalation_via_rogue_named_pipe.toml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+]
+risk_score = 73
+rule_id = "76ddb638-abf7-42d5-be22-4a70b0bf7241"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.action : "Pipe Created*" and
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_via_rogue_named_pipe.toml:33]: What is the `event.category`? Use that would be preferred if possible

Comment (Samirbous) [reply rules/windows/privilege_escalation_via_rogue_named_pipe.toml:33]: image

File: rules/windows/privilege_escalation_via_rogue_named_pipe.toml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Identifies a privilege escalation attempt via rogue named pipe impersonation. An adversary may abuse this technique by
+masquerading as a known named pipe and manipulating a privileged process to connect to it.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Privilege Escalation via Rogue Named Pipe Impersonation"
+note = """## Config
+Named Pipe Creation Events need to be enabled within the Sysmon configuration by including the following settings:
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_via_rogue_named_pipe.toml:18]:
```suggestion

Named Pipe Creation Events need to be enabled within the Sysmon configuration by including the following settings:
```

**Review by brokensound77 — APPROVED** [review]
👍

**Pull request merged**